test/PspUpf.jl

```diff
@@ -135,12 +140,42 @@ end
 
 @testset "PSP energy correction is consistent with fourier-space potential" begin
     q_small = 1e-3    # We are interested in q→0 term
-    for upf_url in upf_urls
+    for upf_url in values(upf_urls)
         psp = load_psp(Downloads.download(upf_url, joinpath(tempdir(), "psp.upf")))
         coulomb = -4π * (psp.Zion) / q_small^2
         reference = eval_psp_local_fourier(psp, q_small) - coulomb
         @test reference ≈ eval_psp_energy_correction(psp, 1) atol=1e-3
     end
 end
 
+@testset "PSP guess density is positive" begin
+    lattice = 5 * I(3)
+    positions = [zeros(3)]
+    for (element, upf_url) in upf_urls
+        psp = load_psp(Downloads.download(upf_url, joinpath(tempdir(), "psp.upf")))
```

Comment:
This is not a good pattern ... Can't we just download them once (using a global variable) and then just use that stored version for the full tests?

Reply:
Yeah, I agree. Ideally, this would all be handled by `load_psp`. Even more Ideally, this would all be in `PseudoPotentialIO` and we'd use the artifact there.

For now though, we can just download them once at the top I think.